### DISCUSSION TITLE
Session expiration time (Day)

### DISCUSSION THREAD

**Discussion by matteobollettini** [rustdesk/rustdesk-server-pro#493] [Q&A] [answered]
Hello everyone

I'm new in RustDesk, so i want to know what "Session expiration time (Day)" means.
Thanks for the support

**Comment by fufesou** [comment, marked as answer]
It is used to set the expiration time in days for your login in 

1. RustDesk client.
2. Web console if "Remember me" is checked.

If the user does not perform any operation within a certain number of days, the cache session will expire(he/she should log in again).

**Reply by matteobollettini** [reply]
Thanks
